src/core/transaction.js

```diff
@@ -1112,6 +1112,77 @@ Transaction.prototype.setDeliverMin = function(deliver_min) {
   return this._setAmount('DeliverMin', deliver_min);
 };
 
+Transaction.prototype.suspendedPayment = function(options_) {
+  const options = _.merge({}, options_);
+
+  if (_.isUndefined(options.cancel_after)) {
```

Comment:
I can't recall the details, but we did agree at some point that method names should be whateverCaseThisIs. rippled itself expects under_scored request properties so this is unavoidable to some degree